src/common/utility.cpp

```diff
@@ -636,4 +636,9 @@ QString Utility::sanitizeForFileName(const QString &name)
     return result;
 }
 
+bool Utility::isConflictFile(const QString &name)
```

Comment:
I'd like to move this wrapper near the wrapped method, so it's easier to find it.